docs/src/main/sphinx/develop/connectors.rst

```diff
@@ -393,13 +403,109 @@ and our ``ConstraintApplicationResult`` can set ``remainingFilter`` to
 This pushdown implementation is quite similar to many Trino connectors
 - see ``MongoMetadata``, ``BigQueryMetadata``, ``KafkaMetadata``.
 
-Let's move on to a more complex example with expression pushdown (a simplified
+Let's move on to a more complex example where the data types from Trino are not
+exactly represented in the underlying database (a simplified
```

Comment:
Far too informal. This is blog post/user guide language, not reference material language. the material in the parenthesis does not make sense and in general, parenthesis should be avoided at all costs.. "Not exactly represented" is confusing phrasing.

Suggest:

The following, more complex example shows data types from Trino that are available directly in the underlying data source, and must be mapped: